File: rest/recording/list-recording-transcriptions/list-recording-transcriptions-5.x.cs
// Download the twilio-csharp library from twilio.com/docs/libraries/csharp
using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account.Recording;

class Example
{
  static void Main(string[] args)
  {
    // Find your Account Sid and Token at twilio.com/console
    const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
    const string authToken = "your_auth_token";

    TwilioClient.Init(accountSid, authToken);

    var transcriptions = TranscriptionResource.Read("REXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX");

    foreach (var transcription in transcriptions)
    {
      Console.WriteLine(transcription.Sid);
      Console.WriteLine(transcription.TranscriptionText);
    }
  }
}

File: rest/recording/list-recording-transcriptions/list-recording-transcriptions-5.x.php
<?php
require_once './vendor/autoload.php';

use Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/console
$sid    = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token  = "your_auth_token";

$twilio = new Client($sid, $token);

$recording = $twilio->recordings("REXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX")
                    ->fetch();

foreach ($recording->transcriptions->read() as $transcription) {
    print($transcription->sid);
    print($transcription->transcriptionText);
}
